tests/erc190/test_package.py

```diff
@@ -39,6 +54,7 @@ def test_ethpm_doesnt_instantiate_with_invalid_package(invalid_package):
         "abcd",
     )
 )
+
```

Comment:
bonus whitespace! (can you remove)

Reply:
Oops..."bonus" makes it sound positive, at least.